distribution/docs/src/main/resources/_contents/_dependency-list/ddf-dependency-list.adoc

```diff
@@ -0,0 +1,286 @@
+* org.eclipse.jetty:jetty-http:jar:9.2.19.v20160908
+* org.eclipse.jetty:jetty-server:jar:9.2.19.v20160908
+* org.eclipse.jetty:jetty-servlet:jar:9.2.19.v20160908
+* org.eclipse.jetty:jetty-util:jar:9.2.19.v20160908
+* org.forgerock.commons:i18n-core:jar:1.4.2
+* org.forgerock.commons:i18n-slf4j:jar:1.4.2
+* org.geotools.xsd:gt-xsd-gml3:jar:8.4
+* org.geotools:gt-cql:jar:13.0
+* org.geotools:gt-cql:jar:8.4
+* org.geotools:gt-epsg-hsql:jar:8.4
+* org.geotools:gt-jts-wrapper:jar:8.4
+* org.geotools:gt-main:jar:8.4
+* org.geotools:gt-opengis:jar:8.4
+* org.geotools:gt-referencing:jar:8.4
+* org.geotools:gt-shapefile:jar:8.4
+* org.geotools:gt-xml:jar:8.4
+* org.glassfish.grizzly:grizzly-framework:jar:2.3.14
+* org.glassfish.grizzly:grizzly-http-server:jar:2.3.17
+* org.hamcrest:hamcrest-all:jar:1.3
+* org.imgscalr:imgscalr-lib:jar:4.2
+* org.jasig.cas:cas-client-core:jar:3.1.10
+* org.jasypt:jasypt:jar:1.9.0
+* org.jcodec:jcodec:jar:0.2.0_1
+* org.jdom:jdom:jar:2.0.2
+* org.joda:joda-convert:jar:1.2
+* org.jolokia:jolokia-osgi:jar:1.2.3
+* org.jscience:jscience:jar:4.3.1
+* org.jvnet.jaxb2_commons:jaxb2-basics-runtime:jar:0.6.0
+* org.jvnet.jaxb2_commons:jaxb2-basics-runtime:jar:0.9.4
+* org.jvnet.ogc:filter-v_2_0_0-schema:jar:1.1.0
+* org.jvnet.ogc:gml-v_3_1_1-schema:jar:1.0.2
+* org.jvnet.ogc:gml-v_3_1_1-schema:jar:1.0.3
+* org.jvnet.ogc:gml-v_3_1_1-schema:jar:1.1.0
+* org.jvnet.ogc:gml-v_3_2_1-schema:jar:1.1.0
+* org.jvnet.ogc:gml-v_3_2_1:pom:1.1.0
+* org.jvnet.ogc:ogc-tools-gml-jts:jar:1.0.3
+* org.jvnet.ogc:ows-v_1_0_0-schema:jar:1.1.0
+* org.jvnet.ogc:ows-v_1_1_0-schema:jar:1.1.0
+* org.jvnet.ogc:wcs-v_1_0_0-schema:jar:1.1.0
+* org.keyczar:keyczar:jar:0.66
+* org.locationtech.spatial4j:spatial4j:jar:0.6
+* org.noggit:noggit:jar:0.6
+* org.objenesis:objenesis:jar:2.1
+* org.openexi:nagasena-rta:jar:0000.0002.0049.0
+* org.openexi:nagasena:jar:0000.0002.0049.0
+* org.opensaml:opensaml-core:jar:3.1.1
+* org.opensaml:opensaml-soap-impl:jar:3.1.1
+* org.opensaml:opensaml-xmlsec-api:jar:3.1.1
+* org.ops4j.pax.exam:pax-exam-container-karaf:jar:4.8.0
+* org.ops4j.pax.exam:pax-exam-junit4:jar:4.8.0
+* org.ops4j.pax.exam:pax-exam-link-mvn:jar:4.8.0
+* org.ops4j.pax.exam:pax-exam:jar:4.8.0
+* org.ops4j.pax.swissbox:pax-swissbox-extender:jar:1.3.1
+* org.ops4j.pax.tinybundles:tinybundles:jar:2.1.1
+* org.ops4j.pax.url:pax-url-aether:jar:2.4.5
+* org.ops4j.pax.url:pax-url-wrap:jar:2.4.5
+* org.osgi:org.osgi.compendium:jar:4.3.1
+* org.osgi:org.osgi.compendium:jar:5.0.0
+* org.osgi:org.osgi.core:jar:4.3.1
+* org.osgi:org.osgi.core:jar:5.0.0
+* org.ow2.asm:asm:jar:5.0.4
+* org.parboiled:parboiled-java:jar:1.1.7
+* org.quartz-scheduler:quartz:jar:2.1.7
+* org.rrd4j:rrd4j:jar:2.2
+* org.slf4j:slf4j-api:jar:1.7.1
+* org.slf4j:slf4j-api:jar:1.7.12
+* org.slf4j:slf4j-ext:jar:1.7.1
+* org.slf4j:slf4j-log4j12:jar:1.7.7
+* org.slf4j:slf4j-simple:jar:1.7.1
+* org.slf4j:slf4j-simple:jar:1.7.5
+* org.spockframework:spock-core:jar:1.0-groovy-2.4
```

Comment:
Should we include dependencies that are used for testing and not part of the final distribution?